packages/api-reference/src/components/Content/Schema/SchemaPropertyHeading.vue

```diff
@@ -40,6 +50,27 @@ const flattenDefaultValue = (value: Record<string, any>) => {
     ? value.default[0]
     : value?.default
 }
+
+// Get model name from schema
+const getModelNameFromSchema = (schema: OpenAPI.Document): string | null => {
+  if (!schema) {
+    return null
+  }
+
+  if (schema.name) {
+    return schema.name
+  }
+
+  if (schemas && typeof schemas === 'object') {
+    for (const [schemaName, schemaValue] of Object.entries(schemas)) {
+      if (JSON.stringify(schemaValue) === JSON.stringify(schema)) {
```

Comment:
wait, those can have circular reference! that will break

Reply:
great catch, fixed